src/design-tokens/mixins.css

```diff
@@ -10,14 +10,11 @@
  * Link button styles
  * 1) display: inline-flex allows us to manage icon placement and the focus background animation
  * 2) Smaller gap between text and icon in link button style
- * 3) 2px padding gives the text breathing room in the focus state
- * 4) 2px margin compensates for the padding so the link doesn't have too much space between it
- *    and surrounding text
- * 5) Has position relative to support the background color pseudo element
- * 6) We hide overflow so the pseudo element can be pushed out of site
- * 7) z-index 0 required for the background to appear correctly on focus
- * 8) Has smaller icons than other button styles
- * 9) Make the focus outline invisible but don't remove it. High contrast mode will remove the background 
+ * 3) Has position relative to support the background color pseudo element
+ * 4) We hide overflow so the pseudo element can be pushed out of site
+ * 5) z-index 0 required for the background to appear correctly on focus
+ * 6) Has smaller icons than other button styles
+ * 7) Make the focus outline invisible but don't remove it. High contrast mode will remove the background 
```

Comment:
Most of the changes in this file are just the numbers being updated since 3 and 4 were moved to the `ClickableStyle` file.